Latex/bilder/vorlage/ba-gc-logo.svg

```diff
@@ -0,0 +1,436 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
kommentar einfach mal hier, damit man drauf antworten kann:

was hast du an der png und svg ded BA-logos geändert? git erkennt die beiden nicht als umbenennungen.
![image](https://user-images.githubusercontent.com/19669567/135264466-09a84a25-99c3-43f0-8d00-21e16df43aec.png)

Reply:
so n Mist, ich hab das extra alles fein per git mv gemacht... bei der PNG is es ja klar aber bei der SVG hätte ich intelligenz erwaretet....
bei beiden stehen jetzt Lizenzinfos drin. bei der SVG die aus Wikipedia, bei der PNG hab ich als Titel was rein geschrieben, Berufsakademie Sachsen als Autor rein genommen und die BA verlinkt